mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-## Unreleased
+## v1.0.2 (September 22, 2023)
+
+### Breaking changes
+* No breaking changes.
+
+### Other changes
+* Building the Mountpoint release on CentOS 7. This gets us compatibilty back to glibc 2.17. This should let mountpoint run on OS which by default packages GCC and Clang that were too old to build CRT and run bindgen respectively in previous release. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
+
+* Updated the CRT submodules to the latest release which is one of the mountpoint-s3 dependencies. This majorly resolves the issue of mountpoint-s3 freeze when trying to upload files to s3 without knowing their full size at the time of upload and there are multiple such file to take time. ([#529](https://github.com/awslabs/mountpoint-s3/pull/529))
```

Comment:
Updated the CRT submodules to the latest releases. This includes bug fixes in `aws-c-s3` library such as [aws-c-s3#347](https://github.com/awslabs/aws-c-s3/pull/347) and [aws-c-s3#346](https://github.com/awslabs/aws-c-s3/pull/346). ([#529](https://github.com/awslabs/mountpoint-s3/pull/529))

Reply:
But changelog should also include what has changed and why customers should care about as recieved review in the last upgrade.

Reply:
For this, I think the main changes are in aws-c-s3 not mountpoint and we can't confirm that it will fix the freeze problem in mountpoint yet.